torchvision/models/inception.py

```diff
@@ -70,10 +70,11 @@ def __init__(self, num_classes=1000, aux_logits=True, transform_input=False):
 
     def forward(self, x):
         if self.transform_input:
-            x = x.clone()
-            x[:, 0] = x[:, 0] * (0.229 / 0.5) + (0.485 - 0.5) / 0.5
-            x[:, 1] = x[:, 1] * (0.224 / 0.5) + (0.456 - 0.5) / 0.5
-            x[:, 2] = x[:, 2] * (0.225 / 0.5) + (0.406 - 0.5) / 0.5
+            x = torch.cat((
+                torch.unsqueeze(x[:, 0], 1) * (0.229 / 0.5) + (0.485 - 0.5) / 0.5,
```

Comment:
FYI: Here I didn't use narrow directly because it seems it is not yet supported by ONNX export.

```
exporting inception_v3 failed:
ONNX export failed: Couldn't export operator aten::narrow
```